packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -351,14 +351,50 @@ export class MgtLogin extends MgtTemplatedComponent {
         light-dismiss
         @opened=${this.flyoutOpened}
         @closed=${this.flyoutClosed}>
-        <div slot="flyout">
-          <fluent-card class="flyout-card">
-            ${this.renderFlyoutContent()}
-          </fluent-card>
-        </div>
+        <fluent-card 
+          slot="flyout" 
+          tabindex="0" 
+          class="flyout-card"
+          @keydown=${this.onUserKeyDown}
+          >
+          ${this.renderFlyoutContent()}
+        </fluent-card>
       </mgt-flyout>`;
   }
 
+  /**
+   * Tracks tabbing through the flyout (keydown)
+   */
+  private readonly onUserKeyDown = (e: KeyboardEvent): void => {
+    if (!this.flyout.isOpen) {
+      return;
+    }
+
+    const el = this.renderRoot.querySelector('.popup-content');
+    const focusableEls = el.querySelectorAll('ul, fluent-button');
+    const firstFocusableEl = el.querySelector('#signout-button') || focusableEls[0];
+    const lastFocusableEl =
+      el.querySelector('#signin-different-account-button') || focusableEls[focusableEls.length - 1];
+
+    if (e.key === 'Tab' && e.shiftKey && firstFocusableEl === e.target) {
+      e.preventDefault();
+      (lastFocusableEl as HTMLElement)?.focus();
+    }
+    if (e.key === 'Tab' && lastFocusableEl === e.target) {
+      e.preventDefault();
+      (firstFocusableEl as HTMLElement)?.focus();
+    }
+
+    if (e.key === 'Tab' && e.shiftKey) {
```

Comment:
I'm not really tracking why we need this block? Shift + Tab is a behavior that the browser natively handles, surely we only need to override the default behavior for Shift + Tab while focus is on the last focusable element inside the pop up?

If focus is on the pop up it self and the user presses Shift + Tab I'd expect that the focus moves to the previous focusable element in the DOM and the pop-up closes, not move to the last element inside the focus trap.

Think of it this way, the fluent card is a fence, when focus moves inside the fence then it's trapped, when it's on the fence it is not yet trapped.

Reply:
In the previous implementation, shift+tab just closed the popup without moving to the previous focusable element. When overriding the default behavior when focus is on the last focusable element, the focus first jumps to the first focusable element before coming back to the previous focusable element which is buggy. I tried to force it to get to just the previous element but that still doesn;t work.